app/helpers/orders_helper.rb

```diff
@@ -38,13 +38,17 @@ def order_owner(owner_guid)
   end
 
   def contact_info(contact_address)
+    contact_address = {} if contact_address.nil?
+
     contact_address['Name'] = contact_name(contact_address)
     contact_address['Phone'] = contact_address.fetch('Phones', {}).fetch('Item', {})
 
     contact_address
   end
 
   def contact_name(contact)
-    [contact['FirstName'], contact['LastName']].reject(&:empty?).join(' ') unless contact.nil?
+    contact = {} if contact.nil?
```

Comment:
If nothing is supplied, this method returns an empty string. I'm not sure what you mean